inc/pagetop-btn/_scss/_pagetop-btn.scss

```
@@ -40,6 +40,16 @@
 		background-size: contain;
 		background-color: transparent;
 		box-shadow: none;
+		// When the user provides explicit width/height via the option,
```

Stylelint (17.11.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)